datatypes/papas/Helix.h

```diff
@@ -0,0 +1,47 @@
+//
+//  Helix.hpp
+//  papas
+//
+//  Created by Alice Robson on 11/05/16.
+//
+//
+
+#ifndef Helix_h
+#define Helix_h
+
+#include "Path.h"
+
+namespace papas {
+/// Decribes helix path of charged particles
+class Helix : public Path {
+public:
+  Helix();
+  ~Helix() = default;
+  Helix(TLorentzVector p4, TVector3 origin, double field, double charge);
+  std::vector<double> polarAtTime(double time) const;
+  double timeAtPhi(double phi) const;
+  double phi(double x, double y) const;
+  double rho() const { return m_rho; }
+  double pathLength(double deltat) const;
+  TVector3 pointFromPolar(const std::vector<double>& polar) const;
+  TVector3 pointAtTime(double time) const override;
+  TVector3 pointAtZ(double z) const;
+  TVector3 pointAtPhi(double phi) const;
+  TVector3 extremePointXY() const { return m_extremePointXY; }
+  TVector3 centerXY() const { return m_centerXY; }
```

Comment:
should return a const ref. please check all classes. When a copy can be avoided, it should be avoided in the first place, as soon as the code as the first version of the interface is written (always difficult and time consuming to do that later on)